app/src/main/java/kth/jjve/moveable/MainActivity.java

```diff
@@ -65,7 +66,7 @@
     private ImageView bluetoothDisabled;
     private ImageView bluetoothEnabled;
 
-    private TextView tempTimeView, tempAccView;
+    private TextView tempTimeView, tempAccView, tempIntRotAcc, tempIntRotGyro;
```

Comment:
Is there a reason why you display both and not just the final rotation?

Reply:
Had to check if the values made sense, it was easier that way, but it's temp